kit/azure/networking/README.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Azure Networking
+summary: |
+  deploys new cloud foundation infrastructure.
+  Add a concise description of the module's purpose here.
+# optional: add additional metadata about implemented security controls
+---
+
+# Azure Networking
```

Comment:
I would add a quick description saying that this is a simple hub example, i.e. it's not connected to an on-premise environment but simply demonstrates the peering part. 

As this is a simplified demo, it also doesn't include firewalling, which a productive implementation would need to have